packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -456,52 +458,67 @@ class _BottomNavigationTile extends StatelessWidget {
         break;
     }
 
-    return Expanded(
-      flex: size,
+    Widget result = InkResponse(
+      onTap: onTap,
+      mouseCursor: mouseCursor,
+      child: Padding(
+        padding: EdgeInsets.only(top: topPadding, bottom: bottomPadding),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            _TileIcon(
+              colorTween: colorTween,
+              animation: animation,
+              iconSize: iconSize,
+              selected: selected,
+              item: item,
+              selectedIconTheme: selectedIconTheme ?? bottomTheme.selectedIconTheme,
+              unselectedIconTheme: unselectedIconTheme ?? bottomTheme.unselectedIconTheme,
+            ),
+            _Label(
+              colorTween: colorTween,
+              animation: animation,
+              item: item,
+              selectedLabelStyle: selectedLabelStyle ?? bottomTheme.selectedLabelStyle,
+              unselectedLabelStyle: unselectedLabelStyle ?? bottomTheme.unselectedLabelStyle,
+              showSelectedLabels: showSelectedLabels ?? bottomTheme.showUnselectedLabels,
+              showUnselectedLabels: showUnselectedLabels ?? bottomTheme.showUnselectedLabels,
+            ),
+          ],
+        ),
+      ),
+    );
+
+    if (item.label != null) {
+      result = Tooltip(
+        message: item.label,
+        preferBelow: false,
+        verticalOffset: selectedIconSize + selectedFontSize,
+        child: result,
+      );
+    }
+
+    result = MergeSemantics(
```

Comment:
Do we still need the MergeSemantics here?

Reply:
I get the following test failure from the bottom navigation tests when I try removing it:

```
The following TestFailure object was thrown running a test:
  Expected: has semantics with label: AC
          Tab 1 of 3 with actions: [SemanticsAction:SemanticsAction.tap] with flags: [
            SemanticsFlag:SemanticsFlag.isSelected,
            SemanticsFlag:SemanticsFlag.isFocusable
          ] with textDirection: TextDirection.ltr
  Actual: SemanticsNode:<SemanticsNode#6(Rect.fromLTRB(0.0, 2.0, 266.7, 54.0), actions: [tap],
flags: [isSelected, isFocusable], label: "AC", textDirection: ltr)>
   Which: label was: AC

When the exception was thrown, this was the stack:
```

Reply:
Oh, you'll also have to undo some of the other semantics changes we did here:

* remove MergeSemantics
* make the semantics widget below (the one with `selected: selected`) a container again
* Remove `container: true` from the semantics widget with `label: indexLabel`

I think that should do it and it makes the tree slightly simpler.

Reply:
Ah yes, that worked, thanks!